src/react-sortable-tree.js

```diff
@@ -155,10 +155,12 @@ class ReactSortableTree extends Component {
     const { instanceProps } = prevState;
     const newState = {};
 
+    let isEqualResult = isEqual(instanceProps.treeData, nextProps.treeData);
```

Comment:
nit: `const`

Reply:
Can we also name it something like `isTreeDataEqual`?

Reply:
Of course!